.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+version: 2
+jobs:
+  centos7: &centos
+    docker:
+      - image: feedforce/ruby-rpm:centos7
+        user: builder
+    working_directory: /home/builder/rpmbuild
+    steps:
+      - checkout:
+          path: ~/ruby-rpm
+      - run:
+          name: Copy SPEC file
+          command: cp ~/ruby-rpm/ruby.spec ./SPECS/
+      - run:
+          name: Build Ruby RPM
+          command: |
+            RUBY_X_Y_Z_VERSION=$(grep "%define \+rubyver" ./SPECS/ruby.spec | awk '{print $3}')
+            RUBY_X_Y_VERSION=$(echo $RUBY_X_Y_Z_VERSION | sed -e 's@\.[0-9]$@@')
+
+            cd ./SOURCES
+            curl -LO https://cache.ruby-lang.org/pub/ruby/$RUBY_X_Y_VERSION/ruby-$RUBY_X_Y_Z_VERSION.tar.gz
+
+            rpmbuild -ba ../SPECS/ruby.spec
+
+            mkdir /tmp/ruby-rpm
+            cp ../RPMS/x86_64/* /tmp/ruby-rpm
+            cp ../SRPMS/* /tmp/ruby-rpm
+      - store_artifacts:
+          path: /tmp/ruby-rpm
+          destination: ruby-rpm
+      - persist_to_workspace:
+          root: /tmp/ruby-rpm
+          paths:
+            - ./*
+
+  centos6:
+    <<: *centos
+    docker:
+      - image: feedforce/ruby-rpm:centos6
+        user: builder
+
+  deploy:
+    docker:
+      - image: circleci/golang
```

Comment:
Why is golang necessary? The [`script/github-release.sh`](https://github.com/koshigoe/ruby-rpm/blob/circleci2.0/script/github-release.sh) did not seem to use golang.

Reply:
I don't know what image is best...

I chose `circleci/golang` because downloading Docker image provided by CircleCI is fast (maybe...).
(I have no reason to choose Golang.)

Please tell me what image to choose. :pray:

Reply:
> Please tell me what image to choose. 🙏

When I looked at the following repository, such official images were not distributed from the CircleCI.

https://github.com/circleci/circleci-images

There is only one of the following options.

* Install the necessary tools (ssh, git, openssl, ...etc) manually using the `alpine` images etc
* Use the `circleci/golang` images

If you choose the latter I think that it is good to write the following comments.

```
deploy:
  docker:
    # Although the Golang is not necessary, the CircleCI official image is used for fast downloading
    - image: circleci/golang
```

Reply:
Thanks.

I added comment.

4e50df1